src/base/ChatApp.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, Fragment } from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
+import NavBar from './NavBar.js';
+import ChatList from '../chatList/ChatList.js';
+import FriendsList from '../friendList/FriendsList.js';
+import Friends from '../data/Friends.json';
+import Chats from '../data/Chats.json';
```

Comment:
특별한 경우(클래스 이름, 컴포넌트 이름)가 아니면 변수는 소문자로 시작하는 경우가 많습니다 ㅎㅎ

```suggestion
import chatData from '../data/Chats.json';
```